{{cookiecutter.project_slug}}/frontend/app/lib/slices/authSlice.ts

```diff
@@ -208,99 +216,99 @@ export const createUserProfile =
 
 export const updateUserProfile =
   (payload: IUserProfileUpdate) =>
-  async (dispatch: Dispatch, getState: () => RootState) => {
-    const currentState = getState()
-    if (loggedIn(currentState) && currentState.tokens.access_token) {
-      try {
-        const res = await apiAuth.updateProfile(
-          currentState.tokens.access_token,
-          payload,
-        )
-        if (res) {
-          dispatch(setUserProfile(res))
+    async (dispatch: Dispatch, getState: () => RootState) => {
```

Comment:
Same here.

Reply:
same thing. looks like whitespace was changed. I'll file a task to run a linter on the repo.